tests/test_app.py

```diff
@@ -148,6 +148,6 @@ def test_inlinei18n_templatetag(locale):
     management.call_command("compilejsi18n")
     template = get_template_from_string(template)
     rendered = template.render(Context({"LANGUAGE_CODE": locale})).strip()
-    assert "var django = globals.django || (globals.django = {});" in rendered
+    assert "django = globals.django || (globals.django = {});" in rendered
```

Comment:
Shouldn't it be `const` instead as you're mentioning it in the description?
```suggestion
    assert "const django = globals.django || (globals.django = {});" in rendered
```

Reply:
Omitting the `var` was a lazy way to preserve backwards compatibility without additional version checks.  Is the var/const difference important for the test result?  I figured the rest of the string would be sufficient to determine whether rendering worked.

Reply:
That is correct, my bad :( Will merge your PR now and issue a release with the updated supported Django versions in the upcoming days. Thanks a lot for your contribution! 🍰